xorg-x11-drv-nvidia.spec

```diff
@@ -49,7 +49,7 @@ Requires:         Xorg >= 1.19.0-3
 
 Requires(post):   ldconfig
 Requires(postun): ldconfig
-Requires(post):   grubby
+Requires(post):   (grubby or sdubby)
```

Comment:
Use /usr/sbin/grubby instead of boolean, our infra (mash) can't handle 'or' in boolean statements.